### PR TITLE
Added Login and Signup page component.

### PR DESCRIPTION
**Pull request by zoyatanweer**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+* {
```
Comment (humzahasan) [inline css/main.css:2]: Is this your reset CSS file ? If yes ? 

Are you not using your CL to get this from there using your own CDN?

File: css/main.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    align-items: center;
+    margin: 1rem auto 1rem 0.5rem;
+}
+.desktop-searchbar{
+    background-color: #e9ecef;
+    border:0px solid #f7f7f7;
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px;
+    font-size: 1.5rem;
+    margin: 5px 15px 5px 5px;
```
Comment (sanjitsarkar) [inline css/main.css]: duplication of margin property

**Review by sanjitsarkar — APPROVED** [review]

File: components/login/login.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+.login-form{
+    height: 34rem;
+    margin-top: 9rem ;
+    margin-left: 8rem;
+}
+.form-heading{
+    margin-top: 5rem;
+}
+.login-email, .login-password{
+    border: 1.5px solid #ced4da;
```
Comment (rutu14) [inline components/login/login.css]: Assign variables to color codes

**Review by rutu14 — APPROVED** [review]

File: css/main.css (hunk 1 above)
Comment (zoyatanweer) [reply css/main.css:2]: I have not included reset css in my cl. Didn't knew about it. Will make the changes.

File: components/login/login.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.2rem;
+    height: 4rem;
+    width: 28rem;
+    padding: 8px;
+}
+.login-btn{
+    background-color: var(--primary-color);
+    border: 2px solid #fb6376;
+    border-radius: 5px;
+    color: #f7f7f7;
```
Comment (zoef-yt) [inline components/login/login.css]: use var for colours

File: components/login/login.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    font-size: 1.6rem;
+    margin:2rem;
+    padding: 10px;
+    width: 9rem;
+}
+.login-btn:hover{
+    background-color: var(--login-btn-bg-color-hover);
+    border: 2px solid #e63961;
```
Comment (zoef-yt) [inline components/login/login.css]: use var for colours

File: components/login/login.css [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+}
+.login-para{
+    font-size: 1.7rem;
+}
+.login-link{
+    color: var(--secondary-color);
+    text-decoration: none;
+}
+.login-link:hover{
+    color: #e63961;
```
Comment (zoef-yt) [inline components/login/login.css]: use var for colours

File: components/signup/signup.css [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    margin-top: 3rem ;
+    margin-left: 12rem;
+    margin-bottom: 3rem;
+}
+.form-heading{
+    margin-top: 3rem;
+}
+.signup-fname, .signup-lname, .signup-email, .signup-password{
+    font-size: 1.2rem;
+    border: 1.5px solid #ced4da;
```
Comment (zoef-yt) [inline components/signup/signup.css]: use var for colours

File: components/signup/signup.css [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+}
+.visibility-eye{
+    height:2rem;
+    width: 2rem;
+    margin:0;
+    padding: 0;
+}
+.signup-btn{
+    background-color: var(--primary-color);
+    border: 2px solid #fb6376;
```
Comment (zoef-yt) [inline components/signup/signup.css]: use var for colours

File: components/signup/signup.css [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+    border-radius: 5px;
+    /* color: #f7f7f7; */
```
Comment (zoef-yt) [inline components/signup/signup.css]: remove commented code.

File: components/signup/signup.css [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+    color:#f0ecec;
```
Comment (zoef-yt) [inline components/signup/signup.css]: use var for colours

File: css/main.css [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+    --primary-color: #f72585;
+    --secondary-color: #ff4d6d;
+    --login-btn-bg-color-hover: #e63961;
+    font-size: 62.5%;
+}
+
+
+body{
+    background-color: #f8f9fa;
+    background-color: #f0ecec;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+    padding:1.5rem;   
+}
+.list-item:hover{
+    font-weight:600;
+    color: var(--primary-color);
+}
+.navigation-container{
+    background-color: var(--primary-color);
+    background-color: #f7f7f7;
+    background-color: #f8f9fa;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 13]
```diff
@@ -0,0 +1,209 @@
...
+    box-shadow: 10px 10px 10px -12px #6c757d;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 14]
```diff
@@ -0,0 +1,209 @@
...
+    font-weight:bold;
+    font-size: 2.5rem;
+    text-align:left;
+}
+.nav-brand{
+    text-align:left;
+    margin:1rem auto 1rem 4.8rem;
+}
+.navigation-container .link{
+    background-color: #f8f9fa;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 15]
```diff
@@ -0,0 +1,209 @@
...
+    align-items: center;
+    margin: 1rem auto 1rem 0.5rem;
+}
+.right-nav-img{
+    height: 3rem;
+    width: 3rem;
+    margin-right: 2.5rem;
+}
+.img-search{
+    background-color: #e9ecef;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 16]
```diff
@@ -0,0 +1,209 @@
...
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+    height: 2.73rem;
+    width: 3rem;
+    margin-right:0;
+    margin-bottom: 0;
+    padding:5px 5px 5px 5px;
```
Comment (zoef-yt) [inline css/main.css]: use padding:5px;

File: css/main.css [hunk 17]
```diff
@@ -0,0 +1,209 @@
...
+    flex-direction: row;
+    justify-content: center;
+}
+.img-form{
+    height:50rem;
+    width: 50rem;
+    object-fit: cover;
+}
+.login-signup-form{
+    background-color: #ffffff;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 18]
```diff
@@ -0,0 +1,209 @@
...
+    text-align: center;
+}
+.form-heading{
+    color: black;
+    font-size: 3rem;
+}
+
+/* footer-section */
+.footer{
+    background-color: #023047;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 19]
```diff
@@ -0,0 +1,209 @@
...
+    color:#f8f9fa;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

File: css/main.css [hunk 20]
```diff
@@ -0,0 +1,209 @@
...
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+}
+.footer-title{
+    font-weight: 400;
+    margin-bottom: 2rem;
+}
+.footer-item:hover{
+    color:#f72585;
```
Comment (zoef-yt) [inline css/main.css]: use var for colours

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Nice images. Good job. Good to merge.

**Pull request merged**